ovh/types_cloud_project_kube_nodepool.go

```diff
@@ -96,13 +98,54 @@ func (s *CloudProjectKubeNodePoolCreateOpts) String() string {
 	return fmt.Sprintf("%s(%s): %d/%d/%d", *s.Name, s.FlavorName, *s.DesiredNodes, *s.MinNodes, *s.MaxNodes)
 }
 
+type EffectType int
+
+const (
```

Comment:
Not a fan of types and constants defined mid-file, but ok

Reply:
fixed tab & space chars
and reorg